internal/rbd/clone.go

```diff
@@ -134,6 +134,9 @@ func (rv *rbdVolume) generateTempClone() *rbdVolume {
 	tempClone.ClusterID = rv.ClusterID
 	tempClone.Monitors = rv.Monitors
 	tempClone.Pool = rv.Pool
+	if rv.RadosNamespace != "" {
```

Comment:
@Madhu-1 do we need this check ? if its not set on `rv.RadosNamespace` , its any way unset .

Reply:
Yes, we need this check as we are generating a new temp volume structure from rv struct?

Reply:
This commit was picked from the PR, keeping it wont be harmful

Reply:
well, there is no reason to have an extra check.

Reply:
updated now PTAL